bcda/service/config_test.go

```diff
@@ -32,6 +33,24 @@ func TestLoadConfig(t *testing.T) {
 	t.Log(cfg.RunoutConfig.String())
 }
 
+func TestIsACODisabled(t *testing.T) {
+	tests := []struct {
+		name     string
+		cmsID    string
+		expected bool
+		cfg      *Config
+	}{
+		{"ACOIsDisabled", "TEST1234", true, &Config{ACOConfigs: []ACOConfig{{patternExp: compileRegex(t, "TEST\\d{4}"), Disabled: true}}}},
+		{"ACOIsEnabled", "TEST1234", false, &Config{ACOConfigs: []ACOConfig{{patternExp: compileRegex(t, "TEST\\d{4}"), Disabled: false}}}},
+		{"ACOIsDoesNotExist", "DNE1234", true, &Config{ACOConfigs: []ACOConfig{{patternExp: compileRegex(t, "TEST\\d{4}"), Disabled: false}}}},
```

Comment:
can you also test that an ACOConfig with no `Disabled` field will be treated with the appropriate default?

Reply:
That is not really possible... since the conf package front loads this. The defaults are only used in that regard with the viper package. This test kind of circumvents all that.

Reply:
Gotcha 👍